osu.Game.Rulesets.Osu/Skinning/TickFollowCircle.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Game.Rulesets.Objects.Drawables;
+using osu.Game.Rulesets.Osu.Objects.Drawables;
+
+namespace osu.Game.Rulesets.Osu.Skinning
+{
+    public abstract class TickFollowCircle : FollowCircle
```

Comment:
No, I was just under the impression that ticking behavior was just for legacy skins. Is the default skin also going to wait to play the break animation until a slider tick arrives Because if so, then there's really no reason for `OnSliderRelease` to exist, just `OnSliderBreak`.

Reply:
I tried that and it looks okay, but logically it feels wrong (since the track area immediately reduces.

So I’d likely keep the default as is for the break animation.